dbld/builddeps

```diff
@@ -177,11 +177,24 @@ function install_pip_packages {
         ubuntu-focal)
             pip3 install --upgrade pip
             ;;
+        devshell)
+            python2 -m pip install --upgrade pip
+            python3 -m pip install --upgrade pip
+            ;;
         *)
             python -m pip install --upgrade pip
             ;;
     esac
-    filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs pip install --ignore-installed -U
+
+    case "${OS_PLATFORM}" in
+        devshell)
+            filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs python2 -m pip install --ignore-installed -U
+            filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs python3 -m pip install --ignore-installed -U
+            ;;
+        *)
+            filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs pip install --ignore-installed -U
+            ;;
+    esac
```

Comment:
I will create install_pip2_packages and install_pip3_packages functions, which will be called directly from the devshell dockerfile.